pkg/sshutil/sshutil.go

```diff
@@ -46,6 +48,25 @@ func DefaultPubKeys() []PubKey {
 	return res
 }
 
+func RemoveHostKeys(sshLocalPort int) error {
```

Comment:
Looks like a misnomer.

Can we call this RemoveKnownHostsEntries()?

Reply:
Sure, done.